hack/jenkins/common.sh

```diff
@@ -419,7 +419,7 @@ fi
 
```

Comment:
lets collect test duration too
if a test duration is taking way longer than normal time
like functiona, test is taking 20 minutes vs 5 minutes

Reply:
In this PR, test duration is now calculated and stored in flake_rates.csv, but reporting is not done. This requires some additional workshopping.